jupyterlab_gcedetails/src/details_widget.tsx

```diff
@@ -76,11 +76,7 @@ export class VmDetails extends React.Component<Props, State> {
     return (
       <span className={STYLES.container}>
         <span
-          className={classes(
-            STYLES.icon,
-            ICON_CLASS,
-            STYLES.interactiveHover
-          )}
+          className={classes(STYLES.icon, ICON_CLASS, STYLES.interactiveHover)}
```

Comment:
I might remove these formatting changes for the other libraries. It's strange that the linter did not fix it for those who edited these files.